include/capo/types.hpp

```diff
@@ -20,6 +20,7 @@ enum class Error {
 	eDeviceFailure,
 	eContextFailure,
 	eInvalidValue,
+	eUnknownFormat,
```

Comment:
This will need a corresponding update to the associated array of strings in impl_al.hpp

Reply:
Why is there no static assert in EnumArray to warn that?

Reply:
I don't think it's possible to: the array size is based on `eCOUNT_`, it's just that you're initializing less than the total elements.

Reply:
I'm thinking... We could add a templated constructor, and static_assert that the number of args passed equals the array size. Would that be acceptable?

Reply:
Yeah, definitely

Reply:
That can be on a separate PR though, I'll add the eUnknownFormat string